libbeat/_meta/fields.ecs.yml

```diff
@@ -1807,6 +1813,7 @@
     - name: bytes
       level: core
       type: long
+      format: bytes
       description: Bytes sent from the source to the destination.
       example: 184
     - name: domain
```

Comment:
Looks like you missed:
* [http.request.body.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1079)
* [http.request.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1090)
* [http.response.body.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1110)
* [http.response.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1121)

Reply:
Thanks, you're right. I'll go add them
🏃